--- a/Cargo.toml
+++ b/Cargo.toml
@@ -67,15 +67,18 @@
 features = ["env-filter"]
 
 [dependencies.turso_core]
+path = "../turso_core"
 version = "0.7.2"
 
 [dependencies.turso_sdk_kit]
+path = "../turso_sdk_kit"
 version = "0.7.2"
 
 [dependencies.turso_sdk_kit_macros]
 version = "0.7.2"
 
 [dependencies.turso_sync_engine]
+path = "../turso_sync_engine"
 version = "0.7.2"
 
 [build-dependencies.bindgen]
