--- a/Cargo.toml
+++ b/Cargo.toml
@@ -41,10 +41,6 @@
 path = "src/lib.rs"
 doc = false
 
-[dependencies.env_logger]
-version = "0.11.6"
-default-features = false
-
 [dependencies.genawaiter]
 version = "0.99.1"
 default-features = false
@@ -59,24 +55,18 @@
 [dependencies.tracing]
 version = "0.1.41"
 
-[dependencies.tracing-appender]
-version = "0.2.3"
-
-[dependencies.tracing-subscriber]
-version = "0.3.20"
-features = ["env-filter"]
-
 [dependencies.turso_core]
+path = "../turso_core"
 version = "0.7.2"
 
 [dependencies.turso_sdk_kit]
+path = "../turso_sdk_kit"
 version = "0.7.2"
 
 [dependencies.turso_sdk_kit_macros]
 version = "0.7.2"
 
 [dependencies.turso_sync_engine]
+path = "../turso_sync_engine"
 version = "0.7.2"
 
-[build-dependencies.bindgen]
-version = "0.69.5"
